pom.xml

```diff
@@ -50,6 +50,7 @@
         <!-- tomcat version &gt; 7.0.56 causes SSL to fail -->
         <tomcat.version>7.0.56</tomcat.version>
         <jackson.version>2.5.3</jackson.version>
+        <jackson.datatype.guava.version>2.4.0</jackson.datatype.guava.version>
```

Comment:
If `jackson.datatype.guava.version` removed then Jackson won't be able to marshal `Optional` from Guava. If I remember correctly, it was needed somewhere (will double check when re-creating PR).
(I understood concern about 1-time variables)